src/RollerPeriphery.sol

```diff
@@ -22,6 +25,36 @@ contract RollerPeriphery is Trust {
     using SafeTransferLib for ERC20;
     using FixedPointMathLib for uint256;
 
+    /* ========== PUBLIC CONSTANTS ========== */
+
+    /// @notice ETH address
+    address public constant ETH = 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE;
+
+    /* ========== PUBLIC IMMUTABLES ========== */
+
+    /// @notice Permit2 contract
+    IPermit2 public immutable permit2;
+
+    // 0x ExchangeProxy address. See https://docs.0x.org/developer-resources/contract-addresses
+    address public immutable exchangeProxy; // TODO: do we want this to be mutable?
```

Comment:
we've decided to keep it as immutable but, answering your question:
if it were mutable, in case that the `exchangeProxy` changes, we could have a setter so we can modify it, so we would be more flexible. 
We believe this is unlikely to happen so we decided to go with the `immutable` variable (which is cheaper gas-wise)